VespaGAS.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
I have a bunch of vague thoughts I might come back to later, but my first question is about conforming to 'Equatable'. I'm assuming neither `Volume`, nor `Gasoline` nor `Oil` conform to it (having an `==` function is not the same as conforming to `Equatable`). Maybe a little code repetition is actually better in this case (i.e. implement `==` for each of them)?